pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -81,6 +81,7 @@ type EditOptions struct {
 	updatedResultGetter func(data []byte) *resource.Result
 }
 
+// editPrinterOptions contains the options of printer
```

Comment:
This is a private a struct. I don't think we need to add comment for this. The name is quite clear for us.

Reply:
I agree with you.  Thanks